packages/core/useMediaQuery/index.ts

```diff
@@ -15,11 +15,20 @@ export function useMediaQuery(query: string) {
     matches.value = event.matches
   }
 
-  mediaQuery.addEventListener('change', handler)
+  const registerEvent = () => {
+    try {
+      mediaQuery.addEventListener('change', handler)
+    }
+    catch (e) {
+      // @ts-ignore
+      // Fallback for Safari < 14 and older browsers
+      mediaQuery.addListener(handler)
+    }
+  }
+
+  registerEvent()
 
-  tryOnUnmounted(() => {
-    mediaQuery.addEventListener('change', handler)
-  })
+  tryOnUnmounted(registerEvent)
```

Comment:
Oh, this should be `removeEventListener`

Reply:
Sorry, can you please give me an example?

Reply:
Oh, I see hehe